curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md

```diff
@@ -7,9 +7,17 @@ dashedName: step-6
 
 # --description--
 
-HTML elements are often nested within other HTML elements. In the previous step you nested the `h2` element, comment and `p` element within the `main` element. A nested element is a child of its parent element.
+In the previous step you nested the `h2` element, comment, and `p` element within the `main` element. A nested element is a child of its parent element and, for readability, should be indented two more spaces than its parent.
 
-To make HTML easier to read, indent the `h2` element, the comment, and `p` element exactly two spaces to indicate they are children of the `main` element.
+Add two spaces in front of the three child elements of `main` so your HTML is more readable.
+
+Indentation in code works like the list tag in html. For example
+```
```

Comment:
- an empty line between line `14` "Indentation in code works like the list tag in html. For example" and line `15` 
- in line `15` add html after triple grave accent